compose/Makefile

```diff
@@ -175,6 +200,27 @@ test-at-behave: test-at-build  ## AMAUAT: run behave, default is `make test-at-b
 			-D transfer_source_path=archivematica/archivematica-sampledata/TestTransfers/acceptance-tests \
 			-D home=archivematica)
 
+test-at-black-box: TAGS=black-box  ## Run the black-box automation tests.
+test-at-black-box: test-at-behave
+
+PY ?= python3
+test-black-box-local:  ## Run the black-box tests without docker. [Options] Python argument as interpreter, e.g `test-black-box-local PY=python2`, default py3
+	virtualenv venv && \
+	. venv/bin/activate && \
+	cd "../src/archivematica-acceptance-tests" && \
+	$(PY) -m pip install -r requirements.txt && \
+	$(PY) -m behave --no-skipped -v --stop -D am_url="http://127.0.0.1:62080/" \
+	       -D am_username="test" \
+	       -D am_api_key="test" \
+	       -D ss_url="http://127.0.0.1:62081/" \
+	       -D ss_username="test" \
+	       -D ss_api_key="test" \
+	       --tags=black-box && \
+	cd "../../compose/" && \
+	deactivate && \
+	rm -r "venv"
```

Comment:
You could leave the env in a location that is not tracked by git so we don't have to recreate on each run. I haven't tested but you could try something like the following:

```Makefile
PY ?= python3
TAGS ?= black-box
ENV ?= $(CURDIR)/.envs.ignoredir/archivematica-acceptance-tests
SRC ?= $(CURDIR)/../src/archivematica-acceptance-tests
test-at-behave-local:  ## Help message... (TAGS, PY)
	@virtualenv --quiet --python=$(PY) $(ENV)
	@pip install --quiet -r $(SRC)/requirements.txt
	$(ENV)/bin/python -m behave \
		--tags=$(TAGS) --other-flags
```

Reply:
Also to be considered, `virtualenv` would need to be listed in the [requirements section](https://github.com/artefactual-labs/am/tree/master/compose#requirements).